.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,7 @@
     Tests are required for bugfixes and new features. Documentation changes
     are necessary for formatting and most enhancement changes. -->
 
-- [ ] Add a CHANGELOG entry if necessary?
+- [ ] Add a CHANGELOG entry if necessary (CHANGES.md) ?
```

Comment:
```suggestion
- [ ] Add a CHANGES.md entry if necessary?
```
Shorter, and better to use only one term

Reply:
thanks

Reply:
I feel like this assumes people know what `CHANGES.md` references, why not this?
```markdown
- [ ] Add an entry in `CHANGES.md` if necessary?
```

Reply:
Sure, that's even better